setup.cfg

```diff
@@ -1,6 +1,5 @@
 [metadata]
 name = qtile
-version = 0.15.0
```

Comment:
Should we replace this by something in setup.py?

Reply:
replaced by setuptools_scm
```
HEAD is now at ba771bab Release 0.15.1
 arnas@zordmachine  ~/out/q/scm  ➦ ba771bab  python setup.py --version
0.15.1
```